db/schema.rb

```diff
@@ -337,6 +337,10 @@
     t.datetime "created_at",                :null=>false
     t.datetime "updated_at",                :null=>false
     t.string   "view_password",             :limit=>255
+    t.boolean  "use_public",                :default=>true
```

Comment:
You also need to commit the new db version number, right at the top of the file. It should match the datetime portion of your migration file.

Reply:
Would it conflict with master? I rebased on master, saw this conflict and kept HEAD's. But i can revert back the version number to my migration's.

Reply:
Louie's one got merged first. Rename yours so the date is later.